grafana-plugin/pkg/plugin/resources.go

```diff
@@ -132,6 +132,7 @@ func (a *App) registerRoutes(mux *http.ServeMux) {
 	//mux.HandleFunc("/debug/settings", a.handleDebugSettings)
 	//mux.HandleFunc("/debug/permissions", a.handleDebugPermissions)
 	//mux.HandleFunc("/debug/stats", a.handleDebugStats)
+	//mux.HandleFunc("/debug/unlock", a.handleDebugUnlock)
```

Comment:
Any reason to add the handler but keep the route commented?

Reply:
Concerned someone will automate API to defeat the whole purpose of having the timeout lock.  On the other hand there are valid use cases when you added some new data and don't want to wait  5 mins.  Maybe it should
always be available under plugin instead?

Currently the endpoint implementation is the minimum for testing, no error handling for when things aren't locked etc.

Reply:
:+1: right; maybe, we can leave it like this for now and revisit later